### DISCUSSION TITLE
Help with class-based AuthChecker

### DISCUSSION THREAD

**Discussion by noahg** [MichalLytek/type-graphql#1139] [Q&A] [answered]
I'm trying to implement the class-based AuthChecker mentioned in the docs but the AuthCheckerInterface is not available in the published package. Also, the BuildContext.authChecker is only typed to the function-based AuthChecker. It feels like I'm missing something obvious. Can someone point me in the right direction, or perhaps this is unsupported at the moment? Thanks.

**Comment by MichalLytek** [comment, marked as answer]
Did you read the docs for `v1.2.0-rc1` but installed `v1.1.1`?

**Reply by noahg** [reply]
Ha, knew it was something obvious. Thanks for the quick response!
